examples/features/error_handling/README.md

```diff
@@ -0,0 +1,22 @@
+# Description
+
+This example demonstrates basic RPC error handling in gRPC.
+
+# Run the sample code
+
+Run the server, which returns an error if the RPC request's `Name` field is
+empty.
+
+```sh
+$ go run ./server/main.go
```

Comment:
I've seen that the shell command in other READMEs in our repo start with a `$`. But when we preview the markdown, there is usually a copy to clipboard option for these codeblocks; which copies the `$` prompt and needs further editing. 

what do you think? Should we start removing them from here on now? Or keep it as is (like the rest of them)

Reply:
If we want to make a change, let's do it as a cleanup PR across all examples.  The problem with not using the `$` is more apparent when there is example output accompanying it.  I think this is minor enough that we don't need to care much either way, and we should aim to be consistent.

Reply:
SGTM